recipes/elephas/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "elephas" %}
+{% set version = "0.4.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5cb28cb23827b45d96f5af59379cd7f739ee9e5785e9b61fe3a22440d4fea528
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+
+  host:
+    - cython
+    - flask
+    - keras
+    - pip
+    - pyspark
```

Comment:
Are pyspark, keras, and flask needed as host deps?

Reply:
@scopatz I just realized that I didn't take elephas out while working on dictlib and that is the recipe you are looking at (elephas) I will take out elephas (for which I am troubleshooting the Windows build) and refresh the PR for dictlib. I am sorry about that.